### ISSUE TITLE
Game is unable to launch due to unicode letters in libgdx filepath

### ISSUE THREAD

**Issue by Dolphinssss** [exch-bms2/beatoraja#609]
When I try to launch beatoraja, the game tries to find a folder inside ``.../AppData/Local/Temp/`` called ``libgdx[name]``
The game creates this folder upon being launched, however, it seems that libgdx or beatoraja is using my real full name, which contains the character Ø. 

To clarify: it is able to CREATE the folder correctly, but is not able to READ from it or locate it.
![beatoraja command line](http://img.keysounds.net/sXSOaiSD.png)

![libgdx folder](http://img.keysounds.net/QNd04SvM.png)

I censored the name to protect my own privacy, but I hope you still understand what is going on here.

I tried simply renaming the folder to "libgdxDolphin" but it did not work. It will just create a new folder with my full name again.
There are similar problems related to unexpected characters in filepaths, but the other ones seem fixable. For this one, I genuinely do not know how to solve it.

**Comment by wcko87** [comment]
have you tried applying the UTF-8 fix?
using japanese locale only fixes issues of this sort if the issues stem from japanese characters - for all other characters, the UTF-8 fix should cover that.

**Comment by Dolphinssss** [comment]
I tried applying it but nothing changed. Same error seemingly.

**Comment by wcko87** [comment]
try adding this line to your bat file:
```
set USERNAME=dolphin
```

So it looks like this:
```
REM *** Set system-wide "_JAVA_OPTIONS" environment variable to use OpenGL pipeline (improved performance of > 30% potentially. Also use anti-aliasing for non-LR2 fonts, and finally allow Swing framework to utilize AA and GTKLookAndFeel for config window. ***
set USERNAME=dolphin
set _JAVA_OPTIONS='-Dsun.java2d.opengl=true -Dawt.useSystemAAFontSettings=on -Dswing.aatext=true -Dswing.defaultlaf=com.sun.java.swing.plaf.gtk.GTKLookAndFeel' -Dfile.encoding="UTF-8"
pushd %~dp0
java -Xms1g -Xmx4g -cp beatoraja.jar;ir/* bms.player.beatoraja.MainLoader
popd
```
This makes libgdx create the libgdx appdata directory with the username dolphin instead.
source: https://steamcommunity.com/app/646570/discussions/1/2561864094351818098/

**Comment by Dolphinssss** [comment]
Oh that's super useful. I'll try it out in a few moments!
in other news, I found out where on earth libgdx is pulling my real name from!

![Windows User Profiles](http://img.keysounds.net/U8huBNCp.png)

For some context: I ran into a similar issue where another software was not able to load its own installation files because they were in my documents folder (not by my own choice). 
When I initially got my current computer, I made the mistake of creating a user with my own real name, because I thought it only affected it login screen, and not the actual name of the user folder. After going through some hoops with registry edits, I was able to forcibly rename both my display name and the Windows local user name (and its associated folder) to "Dolphin", but it seems my real name still stuck around in this one place. If you know where to find it in the registry edit then I can go in and fix that too, since I'm honestly expecting it to cause issues in the future for other software as well.

**Comment by Dolphinssss** [comment]
adding that line fixed the game immediately!!!! i can finally play the game now!

**Comment by wcko87** [comment]
Just documented the fix in my guide discussions page.
https://github.com/wcko87/beatoraja-english-guide/discussions/32

**Comment by potchy** [comment]
There's an open issue in libgdx's repo for this exact problem: https://github.com/libgdx/libgdx/issues/5524
Then one in OpenJDK's issue tracker: https://bugs.openjdk.java.net/browse/JDK-8195129

It seems it's going to be incredibly difficult to get this fixed

**Comment by wcko87** [comment]
yes, but it is also easily fixed on our end so I think it is okay to close this issue
